### PR TITLE
Form validation

### PR DESCRIPTION
**Pull request by pchaussalet**
Bare minimum to support error display and mandatory fields

### CODE REVIEW

**Comment by pchaussalet** [comment]
@joshuarule: you could probably improve the mandatory field style ;)

**Comment by joshuarule** [comment]
I tried a few different options but came to the conclusion that what you have is the safest for handling the various required controls.

File: ui/select.reel/select.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
                 "element": {"#": "selectButton"}
             },
             "bindings": {
-                "disabled": {"<-": "@owner._options.length == 0"}
+                "disabled": {"<-": "@owner.disabled || @owner._options.length == 0"}
```
Comment (thibaultzanini) [inline ui/select.reel/select.html:27]: good catch 👍

**Comment by thibaultzanini** [comment]
LGTM

**Pull request merged**